commit 3ec0e3ea78cf3f1bcf2d6c13170f59a186b2a359 (refs/changes/49/1549/2)
Author: Trent Mick <trentm@gmail.com>
Date:   2017-02-22T11:23:05-08:00 (2 years, 7 months ago)
    
    MANTA-3171 initial minimal doc rebranding updates for Triton Object Storage

diff --git a/README.md b/README.md
index 07bdc26..56751cb 100644
--- a/README.md
+++ b/README.md
@@ -5,14 +5,15 @@
 -->
 
 <!--
-    Copyright (c) 2016, Joyent, Inc.
+    Copyright 2017 Joyent, Inc.
 -->
 
 # Manta: object storage with integrated compute
 
-Manta is an open-source, HTTP-based object store that uses OS containers to
-allow running arbitrary compute on data at rest (i.e., without copying data
-out of the object store).  The intended use-cases are wide-ranging:
+Manta, Triton's object storage and converged analytics solutions, is an
+open-source, HTTP-based object store that uses OS containers to allow running
+arbitrary compute on data at rest (i.e., without copying data out of the object
+store).  The intended use-cases are wide-ranging:
 
 * web assets (e.g., images, HTML and CSS files, and so on), with the ability to
   convert or resize images without copying any data out of Manta
diff --git a/docs/operator-guide/index.md b/docs/operator-guide/index.md
index 37270fd..3cc2d91 100644
--- a/docs/operator-guide/index.md
+++ b/docs/operator-guide/index.md
@@ -1,5 +1,5 @@
 ---
-title: Manta Operator's Guide
+title: Object Storage and Converged Analytics
 markdown2extras: tables, code-friendly
 apisections: .
 ---
@@ -10,13 +10,14 @@ apisections: .
 -->
 
 <!--
-    Copyright (c) 2015, Joyent, Inc.
+    Copyright 2017 Joyent, Inc.
 -->
 
 # Manta Operator's Guide
 
-Manta is an internet-facing object store with in-situ Unix-based compute as a
-first class operation.  The user interface to Manta is essentially:
+Manta, Triton's object storage and converged analytics solution, is an
+internet-facing object store with in-situ Unix-based compute as a first class
+operation. The user interface to Manta is essentially:
 
 * A filesystem-like namespace, with *directories* and *objects*, accessible over
   HTTP
@@ -28,15 +29,16 @@ first class operation.  The user interface to Manta is essentially:
 * Users submit map-reduce *compute jobs* that run arbitrary Unix programs on
   their objects.
 
-Users can interact with Manta through the official Node.js CLI; the Joyent user
-portal; the Node, Python, Ruby, or Java SDKs; curl(1); or any web browser.
+Users can interact with Triton Object Storage through the official Node.js CLI;
+the Joyent user portal; the Node, Python, Ruby, or Java SDKs; curl(1); or any
+web browser.
 
 For more information, see the official [public user
 documentation](http://apidocs.joyent.com/manta/).  **Before reading this
-document, you should be very familiar with using Manta, including both the CLI
-tools and the compute (jobs) features.  You should also be comfortable with all
-the [reference material](http://apidocs.joyent.com/manta/) on how the system
-works from a user's perspective.**
+document, you should be very familiar with using Triton Object Storage,
+including both the CLI tools and the compute jobs features. You should also be
+comfortable with all the [reference material](http://apidocs.joyent.com/manta/)
+on how the system works from a user's perspective.**
 
 
 # Architecture basics
diff --git a/docs/user-guide/index.md b/docs/user-guide/index.md
index 8a3ced0..f1ffb60 100644
--- a/docs/user-guide/index.md
+++ b/docs/user-guide/index.md
@@ -11,9 +11,9 @@ This page is to only use the node.js CLI examples. The curl one is for the web a
 Standard Opening Paragraph
 -->
 
-# Introducing the Joyent Manta Service
+# Manta: Triton's object storage and converged analytics solution
 
-Joyent Manta is a highly scalable, distributed object storage service with integrated compute. Developers can store and process any amount of data at any time where a simple web API call replaces the need for spinning up instances. Joyent Manta Compute is a complete and high performance compute environment including R, Python, node.js, Perl, Ruby, Java, C/C++, ffmpeg, grep, awk and others. Metering is by the second with zero provisioning, data movement or scheduling latency costs.
+Manta, Triton's object storage and converged analytics solution, is a highly scalable, distributed object storage service, with integrated compute that enables the creation analytics jobs (more generally, compute jobs) which process and transform data at rest. Developers can store and process any amount of data at any time where a simple web API call replaces the need for spinning up instances. Manta compute is a complete and high performance compute environment including R, Python, node.js, Perl, Ruby, Java, C/C++, ffmpeg, grep, awk and others. Metering is by the second with zero provisioning, data movement or scheduling latency costs.
 
 This page describes the service and how to get started.  You can also skip
 straight to some [compute examples](job-patterns.html).
@@ -69,10 +69,10 @@ Manta.
 
 # Sign Up
 
-To use Joyent Manta Storage Service, you need a Joyent Cloud account.  If
+To use Triton's object storage, you need a Triton Public Cloud account.  If
 you don't already have an account, you can create one at
 [https://my.joyent.com/signup](https://my.joyent.com/signup). You will
-not be charged for Joyent Manta Storage Service until you use it.
+not be charged for object storage until you use it.
 
 Once you have signed up, you will need to add an SSH public key to your account. Joyent recommends using RSA keys, as the node-manta CLI programs will work with RSA keys both locally, and with the `ssh agent`. DSA keys will only work if the
 private key is on the same system as the CLI, and not password-protected.
@@ -81,9 +81,9 @@ private key is on the same system as the CLI, and not password-protected.
 Standard Closing Paragraph
 -->
 
-# An Integral Part of the Joyent Cloud
+# An Integral Part of the Triton Public Cloud
 
-The Joyent Manta Service is just one of a family of services. Joyent Compute Services range from instances in our standard Persistent Compute Service (metered by the hour, month, or year) to our ephemeral Joyent Manta Compute Service (by the second).  All are designed to seamlessly work with our Object Storage and Data Services.
+The Triton object storage service is just one of a family of services. Triton Public Cloud services range from instances in our standard Persistent Compute Service (metered by the hour, month, or year) to our ephemeral Manta compute service (by the second).  All are designed to seamlessly work with our Object Storage and Data Services.
 
 <!--
 From this point on, this is the same exact material as inside the portal
@@ -124,7 +124,7 @@ programs, it is significantly easier for you to set them globally in your
 environment.  There are four environment variables that all command line tools look for:
 
 * `MANTA_URL` - The API endpoint
-* `MANTA_USER` - Your Joyent Cloud account login name
+* `MANTA_USER` - Your Triton Public Cloud account login name
 * `MANTA_SUBUSER` - A user who has limited access to your account.
 See [Role Based Access Control and Manta](rbac.html)
 * `MANTA_KEY_ID` - The fingerprint of your SSH key.
@@ -132,7 +132,7 @@ See [Role Based Access Control and Manta](rbac.html)
 Copy all of the text below, and paste it into your `~/.bash_profile` or `~/.bashrc`.
 
 	export MANTA_URL=https://us-east.manta.joyent.com
-	export MANTA_USER=$JOYENT_CLOUD_USER_NAME
+	export MANTA_USER=$TRITON_CLOUD_USER_NAME
 	unset MANTA_SUBUSER # Unless you have subusers
 	export MANTA_KEY_ID=$(ssh-keygen -l -f $HOME/.ssh/id_rsa.pub | awk '{print $2}')
 
@@ -141,7 +141,7 @@ to add the text in the clipboard to your file. like this:
 
     $ pbpaste >> ~/.bash_profile
 
-Edit the `~/.bash_profile` or `~/.bashrc` file, replacing `$JOYENT_CLOUD_USER_NAME` with your Joyent Cloud username.
+Edit the `~/.bash_profile` or `~/.bashrc` file, replacing `$TRITON_CLOUD_USER_NAME` with your Triton Public Cloud username.
 
 Run
 
@@ -175,7 +175,7 @@ of this document.
 
 # CLI
 
-This Getting Started guide uses command line tools that are Joyent Manta Storage Service analogs of common Unix tools (e.g. mls == ls). You can find man pages for these tools in the [CLI Utilities Reference](commands-reference.html)
+This Getting Started guide uses command line tools that are Manta analogs of common Unix tools (e.g. mls == ls). You can find man pages for these tools in the [CLI Utilities Reference](commands-reference.html)
 
 # Create Data
 
@@ -260,7 +260,7 @@ creating, listing, and deleting directories:
 
 ## SnapLinks
 
-SnapLinks are a concept unique to the Joyent Manta Storage Service. SnapLinks
+SnapLinks are a concept unique to the Manta service. SnapLinks
 are similar to a Unix hard-link, and because the system is "copy on write," data
 changes are not reflected in the SnapLink. This property makes SnapLinks a very
 powerful entity that allows you to create any number of alternate names and
@@ -594,7 +594,7 @@ asset.
 
 ## Advanced Usage
 
-This introduction gave you a basic overview of Joyent Manta Storage Service: how
+This introduction gave you a basic overview of Manta storage service: how
 to work with objects and how to use the system's compute environment to operate on
 those objects. The system provides many more sophisticated features, including:
 
@@ -703,6 +703,3 @@ Now let's take it up one more level. You can see what's inside a simple node.js
 
 For more details compute jobs see the
 [Compute Jobs Reference documentation](http://apidocs.joyent.com/manta/jobs-reference.html), along with the [default installed software](compute-instance-software.html) and some of our [built-in compute utilities](compute-instance-utilities.html).
-
-
-
diff --git a/docs/user-guide/jobs-reference.md b/docs/user-guide/jobs-reference.md
index 3f02e87..d527068 100644
--- a/docs/user-guide/jobs-reference.md
+++ b/docs/user-guide/jobs-reference.md
@@ -5,12 +5,12 @@ markdown2extras: wiki-tables, code-friendly
 
 # Compute Jobs Reference
 
-This is the reference documentation for jobs.  Unless otherwise specified,
+This is the reference documentation for Manta compute jobs.  Unless otherwise specified,
 the semantics described here are stable, which means that you can expect that
 future updates will not change the documented behavior.  You should avoid
 relying on behavior not specified here.
 
-You should be familiar with Joyent Manta Storage Service before reading this document. To learn the
+You should be familiar with the Manta storage service before reading this document. To learn the
 basics, see [Getting Started](index.html).
 
 
diff --git a/docs/user-guide/rbac.md b/docs/user-guide/rbac.md
index 7a9aed6..d54a5d2 100644
--- a/docs/user-guide/rbac.md
+++ b/docs/user-guide/rbac.md
@@ -39,11 +39,11 @@ getting a Manta object and listing a Manta directory is allowed:
     CAN getdirectory
 
 
-**Users** are login credentials that are associated with your Joyent Cloud
+**Users** are login credentials that are associated with your Triton Public Cloud
 account. While each user name must be unique within an account,
 user names do not need to be globally unique.
 
-If there is a Joyent Cloud account named `bigco`
+If there is a Triton Public Cloud account named `bigco`
 and another one named `littleco`,
 both can have a user named `contractor`.
 
@@ -77,4 +77,3 @@ To learn more see the main [RBAC documentation](https://docs.joyent.com/jpc/rbac
 
 If you want a quick walkthrough of how access control works with Manta,
 see [Getting Started With Access Control](https://docs.joyent.com/jpc/rbac/quickstart).
-
